doc-content/enterprise-only/bpmn/bpmn-subprocesses-ref.adoc

```diff
@@ -25,14 +25,32 @@ An embedded subprocess encapsulates a part of the process. It must contain a sta
 
-
 .Reusable subprocess
-Reusable subprocesses appear collapsed within the containing process.
+Reusable subprocesses appear collapsed within the parent process. To configure a reusable subprocess, select the parent process, click image:getting-started/diagram_properties.png[], and expand *Implementation/Execution*. Set the following properties:
+
+* *Called Element*: The ID of the subprocess that the activity calls and instantiates.
+* *Independent*: If selected, the subprocess is started as an independent process. If not selected, the active subprocess is canceled when the parent process is terminated.
+* *Abort Parent*: If selected, non-independent reusable subprocesses can abort the parent process when there is an error during the execution of the called process instance. For example, when there's an error when trying to invoke the subprocess or when the subprocess instance is aborted. This property is visible only when the *Independant* property is not selected. The following rules apply:
+** If the reusable subprocess is independent, *Abort parent* is not available.
+** If the reusable subprocess is not independent, *Abort parent* is available.
+* *Wait for completion*: If selected, the subprocess continues if the previous process completes or is aborted. This property is selected by default.
+* *Is Async*: Select if the task should be invoked asynchronously and cannot be executed instantly.
+* *Multiple Instance*: Select to execute the subprocess elements a specified number of times. If selected, the following options are available:
+** *MI Execution mode*: Indicates if the multiple instances execute in parallel or sequentially. If set to *Sequential*, new instances are not created until the previous instance completes.
+** *MI Collection input*: Select a variable that represents a collection of elements for which new instances are created. For example, if the size of collection is three, then three instances of the activity are created.
+** *MI Data Input*: Specifies the name of the variable containing the selected element in the collection. The variable name is used to give the multiple instance node access to the variable value.
```

Comment:
"_The variable name is used to give the multiple instance node access to the variable value._"
This is a little bit strange. It sounds like a description of a variable in general. I would personally reformulate the sentence to express that the purpose of the variable is to access elements in the collection.

Reply:
Simplified as suggested. Thank you.